modules/nlu-testing/src/views/full/confusionMatrix.tsx

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash'
+import React, { FC, Fragment, useEffect, useState } from 'react'
+
+import Plots from './plots'
+
+const ConfusionMatrix: FC<any> = props => {
+  const [CF, setCF] = useState({ intent: [], context: [], slot: [], slotCount: [] })
```

Comment:
Why not use a different useState for each of those variables ? It would be much more clear